src/Helius.ts

```diff
@@ -45,14 +46,15 @@ export class Helius {
    * @constructor
    * @param apiKey - API key generated at dev.helius.xyz
    */
-  constructor(apiKey: string, cluster: Cluster = "mainnet-beta", id: string = "helius-sdk") {
+  constructor(apiKey: string, cluster: HeliusCluster = "mainnet-beta", id: string = "helius-sdk") {
     this.apiKey = apiKey;
     this.cluster = cluster;
-    this.endpoint = heliusClusterApiUrl(apiKey, cluster);
-    this.connection = new Connection(this.endpoint);
+    this.endpoints = getEndpoints(cluster);
+    this.connection = new Connection(`${this.endpoints.rpc}?api-key=${this.apiKey}`, "confirmed");
+    this.endpoint = this.connection.rpcEndpoint;
```

Comment:
The `Connection` object from `@solana/web3.js` exposes the rpc endpoint, so we wire `endpoint` up to that for backwards compatibility.

Reply:
Makes sense! Any specific reason as to why you explicitly set the `commitmentOrConfig` to `"confirmed"` instead of leaving it empty?

Reply:
Good catch, I think this was a leftover from my custom Helius wrapper. Removed it.